ncm-nscd/src/main/perl/nscd.pm

```diff
@@ -19,6 +21,24 @@ $NCM::Component::NoActionSupported = 1;
 my $cfgfile='/etc/nscd.conf';
 my $p ='/software/components/nscd';
 
+
+sub _get_nscd_version {
+    my ($self) = @_;
+
+    my $proc = CAF::Process->new ([qw (nscd -V)], log => $self, keep_state => 1);
```

Comment:
keep_state is not valid, it's `keeps_state` according to https://quattor-documentation.readthedocs.io/en/17.8.0/CAF/Process/

Reply:
Thanks, stupid typo.